Sources/SwiftSyntaxMacroExpansion/MacroExpansion.swift

```diff
@@ -20,7 +20,7 @@ import SwiftSyntax
 @_spi(MacroExpansion) @_spi(ExperimentalLanguageFeature) import SwiftSyntaxMacros
 #endif
 
-public enum MacroRole: Sendable {
+public enum MacroRole: String, Sendable {
```

Comment:
I would prefer not to add `String` as the underlying value here. It prevents us from changing this in the future (eg. due to performance considerations) since removing `String` as the underlying value would be API breaking if clients relied on it. 

So, even it it’s annoying, I’d prefer an internal switch that just maps macro roles to their names.